pages/index.js

```diff
@@ -1,3 +1,15 @@
+import styled from 'styled-components';
+
+styled;
+
 export default function Home() {
-  return <div></div>;
+  return (
+    <div>
+      <headline>CryptoCalc</headline>
+    </div>
+  );
 }
+
+const headline = styled.h1`
```

Comment:
I'm not 100% sure on this but I think it's best to start the name of a styled component with an uppercase letter.
So Headline instead of headline.